### PR TITLE
Adding node.dino.icu

### PR DESCRIPTION
**Pull request by TGH75972**
The subdomain node.dino.icu will be used by me for my personal website

### CODE REVIEW

**Review by ansh3108 — CHANGES_REQUESTED** [review]
Looks like you have some syntax error in your pr!

The value, ```dinode.dino.icu``` isn't the redirecting kind of thing, it should point it to your servers where you are hosting your site for eg if you are hosting your website using github it should point to something like this, ```[your github username].github.io``` and on github pages settings, you should point it to ```node.dino.icu```. In case you are hosting your site through github, it might be the solution.

**Comment by Muirrum** [comment]
Hey! Quick request: can you edit your PR description with a few sentences describing the purpose of this subdomain?

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2686,4 +2686,9 @@ internal:
 - ttl: 1
   type: A
   value: 134.209.190.125
+
+node:
+- ttl: 1
```
Comment (Muirrum) [inline hackclub.com.yaml:2691]:
```suggestion
- ttl: 600
```
TTL needs to be bumped

File: hackclub.com.yaml [hunk 2]
```diff
@@ -2686,4 +2686,9 @@ internal:
...
+  type: CNAME
+  value: node.dino.icu.
```
Comment (Muirrum) [inline hackclub.com.yaml:2693]: This doesn't appear to resolve. Where is your site hosted?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 2 above)
Comment (TGH75972) [reply hackclub.com.yaml:2693]: My site is hosted on github only but i want to have a dino.icu domain

Comment (Muirrum) [reply hackclub.com.yaml:2693]: Alright, then this block needs to be moved into the dino.icu.yml file, and the value needs to point to your GitHub pages site.

Comment (TGH75972) [reply hackclub.com.yaml:2693]: Alright, made the changes

File: dino.icu.yaml [hunk 3]
```diff
@@ -417,3 +417,8 @@ xn--ct9h:
 xn--rl8h:
   type: CNAME
   value: maxwofford.github.io.
+
+node:
+- ttl: 1
```
Comment (Muirrum) [inline dino.icu.yaml:422]:
```suggestion
- ttl: 600
```

TTL needs to be bumped to at least 600

File: hackclub.com.yaml [hunk 4]
```diff
@@ -2686,4 +2686,9 @@ internal:
 - ttl: 1
   type: A
   value: 134.209.190.125
+
+node:
```
Comment (Muirrum) [inline hackclub.com.yaml:2690]: This section needs to be removed from hackclub.com.yml

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml [hunk 5]
```diff
@@ -2686,4 +2686,9 @@ internal:
...
+- ttl: 600
+  type: CNAME
+  value: node.dino.icu.
```
Comment (Muirrum) [inline hackclub.com.yaml:2693]:
```suggestion
```

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml [hunk 6]
```diff
@@ -2686,4 +2686,5 @@ internal:
 - ttl: 1
   type: A
   value: 134.209.190.125
+
```
Comment (Muirrum) [inline hackclub.com.yaml:2689]:
```suggestion
```

File: dino.icu.yaml [hunk 7]
```diff
@@ -417,3 +417,8 @@ xn--ct9h:
 xn--rl8h:
   type: CNAME
   value: maxwofford.github.io.
+
+node:
+- ttl: 600
+  type: CNAME
+  value: tgh75972.github.io.
```
Comment (Muirrum) [inline dino.icu.yaml:424]: It doesn't look like there's a github pages site here. Have you published one yet?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: dino.icu.yaml (hunk 7 above)
Comment (TGH75972) [reply dino.icu.yaml:424]: I have, it's at tgh75972.github.io/personal-website

**Review by Muirrum — APPROVED** [review]
LGTM!

**Pull request merged**